change/@fluentui-react-tabster-eab71fef-90f6-4251-97bf-a6fdb14d98bb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31887&commitId=fb5e220d67598113edfa2fa9e821b06a7dfd681a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>